tools/osbuilder/node-builder/azure-linux/README.md

```diff
@@ -0,0 +1,285 @@
+sudo make deploy
+popd
+```
+
+# Run Kata (Confidential) Containers
+
+## Run via CRI or via containerd API
+
+Use e.g. `crictl` (or `ctr`) to schedule Kata(-CC) containers, referencing either the Kata or Kata-CC handlers.
+
+The following sets of commands serve as a general reference for installing `crictl` and setting up some basic CNI to run pods:
+- Install `crictl`, set runtime endpoint in `crictl` configuration:
+
+  `sudo dnf -y install cri-tools`
+
+  `sudo crictl config --set runtime-endpoint=unix:///run/containerd/containerd.sock`
+
+- Install CNI binaries and set a sample CNI config:
+
+  `sudo dnf -y install cni`
+
+  `sudo mv /etc/cni/net.d/99-loopback.conf.sample /etc/cni/net.d/99-loopback.conf`
+
+  ```
+  sudo tee /etc/cni/net.d/10-mynet.conf <<EOF
+  {
+          "cniVersion": "0.2.0",
+          "name": "mynet",
+          "type": "bridge",
+          "bridge": "cni0",
+          "isGateway": true,
+          "ipMasq": true,
+          "ipam": {
+                  "type": "host-local",
+                  "subnet": "10.22.0.0/16",
+                  "routes": [
+                          { "dst": "0.0.0.0/0" }
+                  ]
+          }
+  }
+  EOF
+  ```
+
+  The `10-mynet` configuration file example is derived from: `https://github.com/containernetworking/cni`
+
+- Create a pod manifest (and apply policy), simple example:
+
+  ```
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    namespace: busybox-ns
+    name: busybox
+    uid: busybox-id
+  spec:
+    containers:
+    - image: docker.io/library/busybox:latest
+      name: busybox
+  ```
+
+- Run a Kata or Kata-CC pod with `crictl`:
+
+  `sudo crictl runp -T 30s -r <runc/kata/kata-cc> <path/to/pod.yaml>`
+
+- Decommission pods:
+
+  `sudo crictl pods`
+
+  `sudo crictl stopp <pod_id>`
+
+  `sudo crictl rmp <pod_id>`
+
+Examples with `crictl` for Kata and Kata-CC pods:
+- `sudo ctr image pull --snapshotter=tardev docker.io/library/busybox:latest`
+- `sudo ctr run --cni --runtime io.containerd.run.kata.v2 --runtime-config-path /usr/share/defaults/kata-containers/configuration.toml -t --rm docker.io/library/busybox:latest hello sh`
+- `sudo ctr run --cni --runtime io.containerd.run.kata-cc.v2 --runtime-config-path /opt/confidential-containers/share/defaults/kata-containers/configuration-clh-snp.toml --snapshotter tardev -t --rm docker.io/library/busybox:latest hello sh`
+
+Example with `crictl` for runc pods:
+- `sudo ctr image pull --snapshotter=tardev docker.io/library/busybox:latest`
+- `sudo ctr run --cni --runtime io.containerd.run.runc.v2 -t --rm docker.io/library/busybox:latest hello sh`
+
+For further usage we refer to the upstream `crictl` (or `ctr`) and CNI documentation.
+
+## Run via Kubernetes
+
+If your environment was set up through `az aks create` the respective node is ready to run Kata or Kata Confidential Containers as AKS Kubernetes pods. In any other case, you can set up your own experimental Kubernetes cluster as well.
```

Comment:
I would replace:

"In any other case, you can set up your own experimental Kubernetes cluster as well."

with something similar to:

"Other types of Kubernetes clusters should work as well - but this document doesn't cover how to set-up those clusters."